src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -662,24 +790,70 @@ private static FormulaValue ModInternal(double arg0, double arg1, IRContext irCo
             return new NumberValue(irContext, result);
         }
 
+        public static FormulaValue ModDecimal(IRContext irContext, DecimalValue arg0dv, DecimalValue arg1dv)
+        {
+            decimal arg0 = arg0dv.Value;
+            decimal arg1 = arg1dv.Value;
+            decimal q;
+
+            if (arg1 == 0m)

```

Comment:
>0m [](http://example.com/codeflow?start=24&length=2)

you could use decimal.Zero

Reply:
I could.  I think I prefer the 0m as it looks more like what we do with integer and double.  I like the consistency.